tests/test_integration.py

```diff
@@ -26,17 +28,8 @@ def json(self):
 
 # Stub replacement for requests.get(url)
 def stub(url, params):
-    test_board_id = os.environ.get('TRELLO_BOARD_ID')
-    fake_response_data = None
-    if url == f'https://api.trello.com/1/boards/{test_board_id}/lists':
-        fake_response_data = [{
-            'id': '123abc',
-            'name': 'To Do',
-            'cards': [{'id': '456', 'name': 'Test card'}]
-        }]
-        return StubResponse(fake_response_data)
-
-    raise Exception(f'Integration test stub no mock for url "{url}"')
+    fake_response_data = [{'_id': ObjectId('63556597e561f3969374d358'), 'title': 'wash the floor', 'status:': 'To Do'}]
+    return StubResponse(fake_response_data)
```

Comment:
We should now be able to get rid of this - and the monkeypatch that uses it - entirely since we're no longer mocking API requests